config/bin_version_strings.cfg

```diff
@@ -330,29 +331,37 @@ glibc;;lgpl;"GNU\ C\ Library\ development\ release\ version\ [0-9](\.[0-9]+)+?$"
 libcurl;;mit-style;"^libcurl\/[0-9](\.[0-9]+)+?$";"sed -r 's/libcurl\/([0-9](\.[0-9]+)+?)$/libcurl:\1/'";
+libdbus;;unknown;"libdbus\ [0-9](\.[0-9]+)+?$";"sed -r 's/libdbus\ ([0-9](\.[0-9]+)+?)$/freedesktop:dbus:\1/'";
 libdev;multi_grep;unknown;'"^libdev$"&&"libev:\ loop\ to\ be\ embedded\ is\ not\ embeddable"&&"^[0-9](\.[0-9]+)?+$"';"sed -r 's/([0-9](\.[0-9]+)+?)$/libdev:\1/'";
-libexpat;;unknown;"expat_[0-9]\.[0-9]+\.[0-9]+$";"sed -r 's/expat_([0-9](\.[0-9]+)+?)$/expat:\1/'";
+libexpat;;mit;"expat_[0-9]\.[0-9]+\.[0-9]+$";"sed -r 's/expat_([0-9](\.[0-9]+)+?)$/expat:\1/'";
 libgcrypt;;lgplv2.1;"[Ll]ibgcrypt\ [0-9](\.[0-9]+)+?\ ";"sed -r 's/[Ll]ibgcrypt\ ([0-9](\.[0-9]+)+?)\ .*/libgcrypt:\1/'";
 libgcrypt;;lgplv2.1;"[Ll]ibgcrypt\ [0-9](\.[0-9]+)+?$";"sed -r 's/[Ll]ibgcrypt\ ([0-9](\.[0-9]+)+?)$/libgcrypt:\1/'";
 libgcrypt;;lgplv2.1;"[Ll]ibgcrypt\ [0-9](\.[0-9]+)+?(-[a-z]+)?$";"sed -r 's/[Ll]ibgcrypt\ ([0-9](\.[0-9]+)+?(-[a-z]+)?)$/libgcrypt:\1/'";
 libgcrypt;;lgplv2.1;"^This\ is\ Libgcrypt\ [0-9]\.([0-9]+)+?\ -\ The\ GNU\ Crypto\ Library$";"sed -r 's/This\ is\ [Ll]ibgcrypt\ ([0-9](\.[0-9]+)+?)\ .*/libgcrypt:\1/'";
-libjpeg;;bsd-style;"Independent\ JPEG\ Group.s\ CJPEG,\ version\ [0-9][a-z]\ \ [0-9]+-[A-Za-z]+-[0-9]+$";"sed -r 's/Independent\ JPEG\ Group.s\ CJPEG,\ version\ ([0-9][a-z])\ .*/libjpeg:\1/'";
-libjpeg-turbo;;bsd-style;"^libjpeg-turbo\ version\ [0-9](\.[0-9]+)+?\ \(build\ [0-9]+\)$";"sed -r 's/libjpeg-turbo\ version\ ([0-9](\.[0-9]+)+?)\ .*/libgjpeg-turbo:\1/'";
-liblzma;;public-domain;"liblzma\ [0-9](\.[0-9]+)+?$";"sed -r 's/liblzma\ ([0-9](\.[0-9]+)+?)$/liblzma:\1/'";
+libgmp;multi_grep;unknown;'"^GNU MP assertion failed: %s$"&&"^[0-9](\.[0-9]+)?+$"';"sed -r 's/([0-9](\.[0-9]+)+?)$/gmplib:\1/'";
 libgpiod;;lgpl;"^gpiofind\ \(libgpiod\)\ [0-9](\.[0-9]+)+?$";"sed -r 's/gpiofind\ \(libgpiod\)\ [0-9](\.[0-9]+)+?)$/libgpiod:\1/'";
 libgpiod;;lgpl;"^gpioinfo\ \(libgpiod\)\ [0-9](\.[0-9]+)+?$";"sed -r 's/gpioinfo\ \(libgpiod\)\ [0-9](\.[0-9]+)+?)$/libgpiod:\1/'";
 libgpiod;;lgpl;"^gpiomon\ \(libgpiod\)\ [0-9](\.[0-9]+)+?$";"sed -r 's/gpiomon\ \(libgpiod\)\ [0-9](\.[0-9]+)+?)$/libgpiod:\1/'";
 libsoup;;lgpl;"^libsoup\/[0-9](\.[0-9]+)+?$";"sed -r 's/^libsoup\/([0-9](\.[0-9]+)+?)$/libsoup:\1/'";
-libmicrohttpd.so.12;strict;lgplv2.1;"^[0-9]\.[0-9](\.[0-9]+)+?$";"sed -r 's/^([0-9]\.[0-9](\.[0-9]+)+?)/gnu:libmicrohttpd:\1/'";
 libharfbuzz.so.0;strict;mit;"^[0-9]\.[0-9](\.[0-9]+)+?$";"sed -r 's/^([0-9]\.[0-9](\.[0-9]+)+?)/harfbuzz_project:harfbuzz:\1/'";
-libnss3;multi_grep;unknown;'"nss_DumpCertificateCacheInfo"&&"^[0-9](\.[0-9]+)?+$"';"sed -r 's/([0-9](\.[0-9]+)+?)$/libnss3:\1/'";
+libidn2;multi_grep;unknown;'"^IDN2_UNKNOWN$"&&"^[0-9](\.[0-9]+)?+$"';"sed -r 's/([0-9](\.[0-9]+)+?)$/libidn2:\1/'";
+libjpeg;;bsd-style;"Independent\ JPEG\ Group.s\ CJPEG,\ version\ [0-9][a-z]\ \ [0-9]+-[A-Za-z]+-[0-9]+$";"sed -r 's/Independent\ JPEG\ Group.s\ CJPEG,\ version\ ([0-9][a-z])\ .*/libjpeg:\1/'";
```

Comment:
^ possible?

Reply:
For IDN there is already, for libjpeg, I was already present, I only moved the line, should I add a ^?

Reply:
aja ... let's leave it as it is. I need to check this with the real binary/library